charts/connect/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: connect
-version: 1.9.0
+version: 1.10.0
```

Comment:
**Important note:** With this line change, when we merge this, we will be releasing a new version of the Helm chart.

Reply:
I'll revert it here, and we're going to create another PR for bump chart version